### ISSUE TITLE
The  loss remains at nan：599it [06:49,  1.45it/s]        iters: 600, epoch: 1 | loss: nan         speed: 0.6871s/iter; left time: 9247.2732s

### ISSUE THREAD

**Issue by ZGP1999** [KimMeen/Time-LLM#14]
No matter how the learning rate is adjusted, the  loss remains at nan
![26dd6139dc44ce859efd271a5d92884](https://github.com/KimMeen/Time-LLM/assets/51398472/4f7f0cdc-faad-45fb-8c65-84d3f2ed28e5)
![39127ddbfac96fda07b5070e05c72f6](https://github.com/KimMeen/Time-LLM/assets/51398472/20d996ba-e2da-4614-80f5-61ae01a62d9e)
My training parameters are as follows：
accelerate launch  --num_processes 1 --main_process_port 4096 run_main.py --task_name long_term_forecast --is_training 1 --root_path ./dataset/ --data_path ETTh1.csv --model_id ETTh1_512_96 --model TimeLLM --data ETTh1 --features M --seq_len 96 --label_len 48 --pred_len 96 --factor 3 --enc_in 7 --dec_in 7 --c_out 7 --des Exp --itr 1 --d_model 32 --d_ff 128 --batch_size 4 --learning_rate 0.001 --llm_layers 32 --train_epochs 1 --model_comment TimeLLM-ETTh1

**Comment by Flipped-May** [comment]
I meet this errror, why the loss remains at nan

**Comment by ZGP1999** [comment]
I think you didn‘t loade the model correctly, and I recommend that you download the model locally.
